js/renovation/icon.tsx

```diff
@@ -23,7 +23,7 @@ export class IconProps {
   defaultOptionRules: null,
   view: viewFunction,
 })
-export default class Icon extends JSXComponent<IconProps> {
+export default class Icon extends JSXComponent<IconProps>(IconProps) {
```

Comment:
Since v1.0.72 generator supports the following notation:
```typescript
JSXComponent(IconProps)
```

Reply:
Ok, thank you!